gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -1033,3 +1030,99 @@ message Shape {
   // The following extension IDs are reserved for private use by any organization.
   extensions 9000 to 9999;
 }
+
+// Describes a stop which is served by trips. All fields are as described in the GTFS-Static specification.
+message Stop {
+  enum WheelchairBoarding {
+    UNKNOWN = 0;
+    AVAILABLE = 1;
+    NOT_AVAILABLE = 2;
+   }
+
+  required string stop_id = 1;
+  optional TranslatedString stop_code = 2;
+  optional TranslatedString stop_name = 3;
+  optional TranslatedString tts_stop_name = 4;
+  optional TranslatedString stop_desc = 5;
+  optional float stop_lat = 6;
+  optional float stop_lon = 7;
```

Comment:
I'm not necessarily sure this is a good idea, but since you are using `TranslatedString` here (aka using GTFS-RT modeling concepts), would it make sense to use `Position` for the stop location as well?  I know that comes with additional fields that don't necessarily make sense for a stop and maybe that's enough to reject the idea, but I felt compelled to ask.

Reply:
I'm a neutral on this, I do see a small benefit and small chance of confusion if we do this